src/gui/tray/ActivityItem.qml

```diff
@@ -0,0 +1,288 @@
+import QtQml 2.12
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import Style 1.0
+
+MouseArea {
+    id: activityMouseArea
+    enabled: (path !== "" || link !== "")
+    width: Style.trayWindowWidth
+    height: Style.trayWindowHeaderHeight
+    Layout.alignment: Qt.AlignLeft
```

Comment:
please put sizing properties where the item is used
use `width` and `height` combined with `Layout.fillWidth` and `Layout.fillHeight` is redundant and contradictory
looks like you really want to keep the `Layout.fillHeight` but I am not sure for `height` versus `Layout.fillWidth`